api/src/main/java/org/openmrs/module/kenyaemr/form/velocity/EmrVelocityFunctions.java

```diff
@@ -234,6 +248,14 @@ public String location() {
 		else
 			return Context.getObsService().getObservationsByPersonAndConcept(p, getConcept(conceptIdentifier));
 	}
+	public String getFacilityType() {
+		AdministrationService administrationService = org.openmrs.api.context.Context.getAdministrationService();
+		GlobalProperty globalProperty = administrationService.getGlobalPropertyObject("kenyaemr.isKDoD");
+		if (globalProperty.getValue() != null) {
+			return globalProperty.getPropertyValue();
+		}
+		return "true";
```

Comment:
Shouldn't this be null by default? Returning a true will mess up things for non-DOD facilities